tests/Feature/Api/V1/AchievementOfTheWeekTest.php

```diff
@@ -37,7 +38,7 @@ public function testGetAchievementOfTheWeek(): void
         /** @var Game $game */
         $game = Game::factory()->create(['ConsoleID' => $system->ID]);
         /** @var Achievement $achievement */
-        $achievement = Achievement::factory()->create(['GameID' => $game->ID]);
+        $achievement = Achievement::factory()->create(['GameID' => $game->ID, 'Flags' => AchievementType::OfficialCore]);
```

Comment:
There's a `published` scope for this.
```
$achievement = Achievement::factory()->published()->create(['GameID' => $game->ID]);
```

Reply:
Addressed in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/c384be10eec370633b6d37d8f2c33714547b872f